scripts/plots/cam/exevs_cam_nam_firewxnest_grid2obs_plots.sh

```diff
@@ -199,8 +199,10 @@ if [ -d $log_dir ]; then
 fi
         
 if [ $SENDCOM = "YES" ]; then
- mkdir -m 775 -p $COMOUTplots
- cpreq evs.plots.${MODELNAME}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
+ if [ -s evs.plots.${MODELNAME}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar ]; then
+  mkdir -m 775 -p $COMOUTplots
```

Comment:
@PerryShafran-NOAA , can you explain why this line is needed? $COMOUTplots is created at line 99 of `jobs/JEVS_CAM_PLOTS`

Reply:
It was already there.  But it is indeed not needed.  Should I remove it?

Reply:
@PerryShafran-NOAA Yes, please! Remove whichever line isn't needed (maybe the second)? @ShelleyMelchior-NOAA

Reply:
Done.